contrib/semgrep.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+
+- id: inline-from-impls
+  patterns:
+    - pattern: |-
+        impl From<$F> for $T { fn from(...) -> $SELF { ... } }
+    - pattern-not-regex: "inline"
```

Comment:
`'\#\[inline\]'` or something like that (to only match `#[inline]`) would be "tighter"

Reply:
Ah yes, nice. Will use, thanks.